src/provider/view/components/Header/Header.tsx

```diff
@@ -1,40 +1,55 @@
 import * as React from 'react';
 
-import {GroupingType, Actionable} from '../../containers/NotificationCenterApp';
+import {GroupingType} from '../../utils/Grouping';
+import {Actionable} from '../../../store/Actions';
+import {CircleButton} from '../CircleButton/CircleButton';
 import {ToggleVisibility} from '../../../store/Actions';
 
-interface HeaderProps extends Actionable {
+import {ClearAllPrompt} from './ClearAllPrompt';
+
+import './Header.scss';
+
+interface Props extends Actionable {
+    visible: boolean;
```

Comment:
Can we name this `centerVisible`, as in `ClearAllPrompt`?

Makes it clearer that it's the same variable, and removes any confusion about this being about showing/hiding the header component.